Packages/com.unity.inputsystem/CHANGELOG.md

```diff
@@ -24,6 +24,7 @@ however, it has to be formatted properly to pass verification tests.
   * In our own measurements, `InputUser.OnEvent` is >9 times faster than before and `RebindingOperation.OnEvent` is ~2.5 times faster.
 - Fixed PS4 controller not recognized on Mac when connected over Bluetooth ([case 1286449](https://issuetracker.unity3d.com/issues/input-system-dualshock-4-zct1e-dualshock-2-v1-devices-are-not-fully-recognised-over-bluetooth)).
 - Fixed restart prompt after package installation not appearing on Unity 2020.2+ ([case 1292513](https://issuetracker.unity3d.com/issues/input-system-after-package-install-the-update-slash-switch-and-restart-prompt-does-not-appear)).
+- Fixed input system not registering multiple inputs ([case 1295535](https://issuetracker.unity3d.com/issues/input-system-not-registering-multiple-inputs)).
```

Comment:
```
- Fixed action with multiple bindings getting stuck in `Performed` state when two or more controls are pressed at the same time ([case ...](...).
  * Regression introduced in 1.1-preview.2.
```

CHANGELOG entries should try to make it as easy as possible for the user to figure out whether a change is relevant or not.

Reply:
Ok, fixed